File: 06_Greatest among 10 numbers/gratest_number.py
# Method 1 to solve this problem.

numLis = [int(input("Enter number: "))for i in range(0, 10)]
print("Largest number you entered is:", sorted(numLis)[9])


# Method 2 to solve this problem.

numLis = [int(input("Enter number: "))for i in range(0, 10)]

for j in range(0, 10):
    if(numLis[0] <= numLis[j]):
        temp = numLis[j]
print("Largest number you entered is:", temp)

File: 06_Greatest among 10 numbers/gratest_number.c
#include<stdio.h>

int main()
{
    int temp=0, numLis[10];

    printf("Enter 10 number of your choice: ");

    for(int i=0; i<=9; i++){
        scanf("%d", &numLis[i]);
    }

    for(int j=0; j<=9; j++){
        if(numLis[0] <= numLis[j]){
            temp = numLis[j];
        }
    }

    printf("Gratest number is: %d", temp);
    return 0;
}
